### PR TITLE
Convert test into automated preview with vite build

### PR DESCRIPTION
**Pull request by spenserblack**
This converts `test.html` into an automated Vercel deployment. Users can still test locally with `cd vercel && npm run dev`, but each pull request should also have an automated preview, now. The preview will also be rebuild with each commit to master.

## How to deploy to vercel

1. Create a Vercel account and link it to GitHub
2. Click "New Project" and select sakura
3. Use Vite as the framework preset
4. Set the root directory to `vercel/`
5. Override the build command to `npm install && cd .. && npm install` (we need to build the CSS just once)

This could be part of the same `package.json` instead of in a separate sub-package, but I didn't want to clutter the main project too much.

## Sample

https://sakura-css-spenserblack.vercel.app/

******

For https://github.com/oxalorg/sakura/issues/77#issuecomment-1190441201
Closes #95

### CODE REVIEW

**Comment by spenserblack** [comment]
You could probably make the production deployment the page on your personal site, too.

File: vercel/index.html [hunk 1]
```diff
@@ -4,29 +4,14 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>HTML5 Test Page</title>
-    <link rel="stylesheet" href="css/normalize.css" />
-    <link id="sakura-css" rel="stylesheet" href="css/sakura.css" />
+    <style id="sakura-css"></style>
```
Comment (oxalorg) [inline vercel/index.html:7]: On the final rendered pages, `style.css` is being added below sakura styles. This leads to `normalize.css` overriding some of sakura. Can we put sakura below index.css
image

**Comment by spenserblack** [comment]
:thinking: My Vercel deployment is working, I wonder why this one fails to build.

**Comment by spenserblack** [comment]
@oxalorg ping for this and #95 

Let me know if I can help somehow.

**Comment by oxalorg** [comment]
Hello @spenserblack, really really appreciate your PR. For some reason it wasn't working as expected, and I thought a flat project structure would be more cleaner.

So I took your preview.js code and built a more flat PR from it here: #113 

I have also added you to the contributors section: https://github.com/oxalorg/sakura/commit/b8b60c352c65503db9ccd26a4e9062385b3d99ce

Thanks a lot for your contributions and keeping patience with me. Took me a long time to get around to this. 💚 🌸 🥳

**Comment by spenserblack** [comment]
No problem, glad it's working!

**Pull request closed without merging**